scanner/updater/export.go

```diff
@@ -18,19 +18,19 @@ import (
 	"github.com/stackrox/rox/scanner/updater/rhel"
 	"golang.org/x/time/rate"
 
-	// default updaters
+	// Default updaters. This is required to ensure updater factories are set properly.
 	_ "github.com/quay/claircore/updater/defaults"
 )
 
 // Export is responsible for triggering the updaters to download Common Vulnerabilities and Exposures (CVEs) data
-// and then outputting the result as a zstd-compressed file with .ztd extension
+// and then outputting the result as a zstd-compressed file named vulns.json.zst.
 func Export(ctx context.Context, outputDir string) error {
 	err := os.MkdirAll(outputDir, 0700)
 	if err != nil {
 		return err
 	}
 	// create output json file
-	outputFile, err := os.Create(filepath.Join(outputDir, "output.json.zst"))
+	outputFile, err := os.Create(filepath.Join(outputDir, "vulns.json.zst"))
```

Comment:
Do we have to update the hard-coded name somewhere else?

Reply:
once merged, I imagine vuln updates will fail since the data will be in a different location, but once the scanners are updated to the latest version, this should no longer be an issue.

when it comes to hardcoding, we'll need to make sure https://github.com/stackrox/stackrox/pull/9409 accounts for this. I also need to update the default config, so thanks for the reminder

Reply:
I have updated https://github.com/stackrox/stackrox/pull/9409 accordingly @jvdm